src/device/pciexp_device.c

```diff
@@ -685,21 +686,24 @@ static void pciexp_tune_dev(struct device *dev)
 	if (!root_cap)
 		return;
 
-	/* Check for and enable Common Clock */
-	if (CONFIG(PCIEXP_COMMON_CLOCK))
-		pciexp_enable_common_clock(root, root_cap, dev, cap);
+	if ((CONFIG(VENDOR_PCENGINES) && dasharo_apu_pcie_pm_enabled()) ||
+	    CONFIG(BOARD_PCENGINES_APU5)) {
```

Comment:
The PCIe device tuning must happen also on other board than PC Engines. So:

```
	if (((CONFIG(VENDOR_PCENGINES) && dasharo_apu_pcie_pm_enabled()) || CONFIG(BOARD_PCENGINES_APU5)) || !CONFIG(VENDOR_PCENGINES) )
```	

If I didn't mess the logic up

Reply:
> The PCIe device tuning must happen also on other board than PC Engines

Thanks, that's what I was going for but wrote something else.  This should be fine: https://github.com/Dasharo/coreboot/compare/a285e7c2f4682730bda5ac1b5241438df0d37fd3..b8ed6ba44c3eb5676cf3a7b8a9797792ff733a42#diff-b186f80942e9926a5ff666f94c3003d8ac56e677f5b00d2881b9079376a7c1a9